packages/manager/modules/pci/src/components/project/offer/badge/badge.scss

```diff
@@ -0,0 +1,6 @@
+pci-project-offer-badge {
```

Comment:
you mean .pci-project-offer-badge?

Reply:
No `pci-project-offer-badge` is the name of the component so this is meant to scope any style added to this component